src/openforms/js/lang/en.json

```diff
@@ -464,6 +464,11 @@
     "description": "Manage process vars component column title",
     "originalDefault": "Field"
   },
+  "ENyg8G": {
+    "defaultMessage": "There are multiple co-sign components specified in this form. At most one should be present.",
+    "description": "Too many cosign components",
```

Comment:
Inconsistent spelling of 'cosign' and 'co-sign' in project :-) if this isn't lingo, then make consistent.

Reply:
yeah, luckily this is only in the message description which is internal (and based on that, the message ID is generated). I've checked an all public-facing terms seem to be spelled `co-sign`.

Both spellings appear to be correct though, making it even more confusing :sweat_smile: